src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,95 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+/**
+ * UI of the application.
+ */
+public class Ui {
+
+    protected Scanner sc;
+
+    /**
+     * Constructs an User Interface.
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Prints a top horizontal line before every executed command.
+     */
+    public void showTopLine() {
+        System.out.println("\n    ________________________________________________________");
+    }
+
+    /**
+     * Prints a bottom horizontal line after every executed command.
+     */
+    public void showBottomLine() {
```

Comment:
Can it be `showLine` instead, considering both of them are doing exactly the same thing?